### PR TITLE
"How to" doc for post deployment

### PR DESCRIPTION
**Pull request by galiacheng**
Including:

* Create WebLogic cluster with advanced configuration -- mainTemplate.md

* Update WebLogic cluster with advanced configuration -- updateClusterTemplate.md

* Configure a database connection -- dbTemplate.md

* Update Java EE application -- updateAppTemplate.md

### CODE REVIEW

**Review by edburns — APPROVED** [review]

File: _includes/sub-template-advanced-usage.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Oracle and Microsoft maintain an [Azure Marketplace offer](https://portal.azure.com/?feature.customPortal=false#create/oracle.20210620-wls-on-aks20210620-wls-on-aks) that 
+makes it easy to get started with WebLogic Server on Azure.  For complete documentation on the offer, see [the user guide](https://oracle.github.io/weblogic-kubernetes-operator/userguide/aks/). 
+
+If you need to go beyond the capabilities of the Azure Marketplace offer, this guidance enables several advanced features.  The following features and more are possible.
+
+- Customize Azure Container Insights with specified retention days, workspace SKU and resource permissions.
+
+- Customize Azure Kubernetes Service version and Agent Pool name.
+
+- Create custom T3 channel for WebLogic Administration Server and cluster and expose the T3 channel via Azure Standard Load Balancer service.
```
Comment (rosemarymarano) [inline _includes/sub-template-advanced-usage.md:10]: WebLogic Administration Server -> WebLogic Server Administration Server

File: _includes/sub-template-create-update-wls-on-aks.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
+## Prepare the Parameters
+
+You must construct a parameters JSON file containing the parameters to be passed to the ARM template.
+For background information about parameter files, see [Create Resource Manager parameter file](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/parameter-files). 
+This section shows how to obtain the values for the required properties.
+
+### Obtain parameter values from Azure portal
+
+The following steps use the [Azure Create UI Definition Sandbox](https://portal.azure.com/?feature.customPortal=false#blade/Microsoft_Azure_CreateUIDef/SandboxBlade) to obtain the values. If you prefer to edit a json file, you can also create the objects in your parameters file directly. The Azure UI Definition Sandbox provides controls to select resources and input your values easily.  **More importantly, this approach generates syntactically valid JSON, eliminating an important class of data entry error.**
```
Comment (rosemarymarano) [inline _includes/sub-template-create-update-wls-on-aks.md:9]: json file -> JSON file

File: _includes/sub-template-dnszone.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+### Custom DNS (optional)
+
+#### Registered Domain Name
+
+You need to buy a domain name to create a custom DNS alias.
+
+#### Azure DNS Zone
+
+If you create the DNS alias on an existing [Azure DNS Zone](https://docs.microsoft.com/en-us/azure/dns/dns-overview), make sure you have perfomed the [Azure DNS Delegation](https://docs.microsoft.com/en-us/azure/dns/dns-domain-delegation).  Once you have completed the delegation, you can verify it with `nslookup`.  For example, assuming your domain name is **contoso.com**, this output shows a correct delegation.
```
Comment (rosemarymarano) [inline _includes/sub-template-dnszone.md:15]: Once you have completed ->  After you have completed

File: _includes/sub-template-prerequisites-storage.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+### Azure Storage account 
+
+If you are deploying a Java EE application or using your own JDBC datasource driver, you are required to
```
Comment (rosemarymarano) [inline _includes/sub-template-prerequisites-storage.md:3]: datasource  -> data source (two words)

File: _includes/sub-template-prerequisites-storage.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+### Azure Storage account 
+
+If you are deploying a Java EE application or using your own JDBC datasource driver, you are required to 
+have application packages and jdbc libraries uploaded to a blob storage container in an Azure Storage Account.
```
Comment (rosemarymarano) [inline _includes/sub-template-prerequisites-storage.md:4]: jdbc  -> JDBC

File: _includes/sub-template-prerequisites-storage.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+To create Azure Storage Account and blobs, follow the steps in [Quickstart: Upload, download, and list blobs with the Azure portal](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal).
+
+Upload your application packages (.jar, .war, .ear files) to the blob.
+
+Upload your jdbc drivers (.jar files) to the blob.
```
Comment (rosemarymarano) [inline _includes/sub-template-prerequisites-storage.md:10]: jdbc -> JDBC

File: aks/dbTemplate.md [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+
+{% include variables.md %}
+
+# Apply Database ARM Template to existing {{ site.data.var.wlsFullBrandName }}
+
+This page documents how to configure an existing deployment of {{ site.data.var.wlsFullBrandName }} with an existing Azure database using Azure CLI. 
+
+You can invoke the database ARM template to:
+
+  - Create a new datasource connection, you can have multiple datasource connections in your cluster.
```
Comment (rosemarymarano) [inline aks/dbTemplate.md:14]: datasource -> data source (two words, please fix globally)

File: aks/dbTemplate.md [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+
+  - Update an existing datasource connection
+
+  - Delete an existing datasource connection
+
+## Prerequisites
+
```
Comment (rosemarymarano) [inline aks/dbTemplate.md:21]: Need intervening text here, in between two headings.

File: aks/dbTemplate.md [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+  - Create your database server, and make sure the database is accessible from Azure.
+
+  - Ship your database driver to WebLogic cluster. The only approach to upload a datasource driver is using [marketplace offer](https://portal.azure.com/?feature.customPortal=false#create/oracle.20210620-wls-on-aks20210620-wls-on-aks). The offer enables you to bring your own datasource driver. See [WebLogic Server Instance](#webLogic-server-instance)
+
+  - Invoke the dbTemplate to update an existing datasource connection or create another new datasource connection.
+
+### Apply multiple datasource
+
+You may want to enable multiple datasource in your cluster for the following usage:
```
Comment (rosemarymarano) [inline aks/dbTemplate.md:66]: enable multiple datasource -> enable multiple data sources

File: aks/updateAppTemplate.md [hunk 10]
```diff
@@ -0,0 +1,260 @@
...
+
+
+### Java EE application location
+
+The template supports two approach to specify the location of Java EE application. 
+The template will update the cluter with applications specified in `appPackageUrls` and `appPackageFromStorageBlob`.
+
+#### SAS Urls
+
+You can specify the application urls via `appPackageUrls`. The template only supports url from Azure Storage Account.
```
Comment (rosemarymarano) [inline aks/updateAppTemplate.md:70]: urls  -> URLs (globally)

File: aks/updateClusterTemplate.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+
+This document will guide you to update a WebLogic cluster using the advanced configurations.
+
+## Prerequisites
+
+{% include sub-template-prerequisites.md %}
+
+## Updating the existing WebLogic Server cluster
+
+The template will apply the new configuration in `parameters.json` to the running WebLogic cluster, please double check you have specified:
```
Comment (rosemarymarano) [inline aks/updateClusterTemplate.md:32]: double check you have specified ->  double check that you have specified

**Review by rosemarymarano — COMMENTED** [review]
Some edits. It would be easier for me to see/review these docs on a live site. Let me know if you want me to do that.

File: aks/updateClusterTemplate.md (hunk 11 above)
Comment (galiacheng) [reply aks/updateClusterTemplate.md]: Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb

File: aks/dbTemplate.md (hunk 9 above)
Comment (galiacheng) [reply aks/dbTemplate.md]: Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb

File: aks/dbTemplate.md (hunk 8 above)
Comment (galiacheng) [reply aks/dbTemplate.md:21]: Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb

File: _includes/sub-template-prerequisites-storage.md (hunk 6 above)
Comment (galiacheng) [reply _includes/sub-template-prerequisites-storage.md]: Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb

File: _includes/sub-template-prerequisites-storage.md (hunk 5 above)
Comment (galiacheng) [reply _includes/sub-template-prerequisites-storage.md]: Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb

File: _includes/sub-template-prerequisites-storage.md (hunk 4 above)
Comment (galiacheng) [reply _includes/sub-template-prerequisites-storage.md]: Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb

File: _includes/sub-template-dnszone.md (hunk 3 above)
Comment (galiacheng) [reply _includes/sub-template-dnszone.md]: Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb

File: _includes/sub-template-create-update-wls-on-aks.md (hunk 2 above)
Comment (galiacheng) [reply _includes/sub-template-create-update-wls-on-aks.md]: Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb

File: _includes/sub-template-advanced-usage.md (hunk 1 above)
Comment (galiacheng) [reply _includes/sub-template-advanced-usage.md]: Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb

**Review by mriccell — APPROVED** [review]

**Pull request merged**